lib/tokenize.js

```diff
@@ -5,10 +5,24 @@
  *
  * @return  {array}
  */
-module.exports = function (input) {
-    return input
-        .toLowerCase()
-        .replace(/[^a-z0-9á-úñäâàéèëêïîöôùüûœç\-\' ]+/g, '')
-        .replace('/ {2,}/',' ')
-        .split(' ');
+var spliddit = require('spliddit');
+
+module.exports = function(input) {
+    if (input === undefined) return [];
```

Comment:
Because we check if the input if `undefined` in the containing module (https://github.com/thisandagain/sentiment/blob/develop/lib/index.js#L42), this can be removed.